modules/transport.js

```diff
@@ -374,7 +374,7 @@ Transport.prototype.onNewBlock = function (block, broadcast) {
 	if (broadcast) {
 		modules.system.update(function () {
 			if (!__private.broadcaster.maxRelays(block) && !modules.loader.syncing()) {
-				modules.peers.list({}, function (err, peers) {
+				modules.peers.list({normalized: false}, function (err, peers) {
```

Comment:
Can we make `normalized: false` default, so no need to pass that param every time?